docs/UserGuide.md

```diff
@@ -370,14 +390,19 @@ please use [List Recipes](#436-list-recipes) command.
 
 **Additional Information:**<br />
 - Current requirements for a keyword:
```

Comment:
These are not requirements of keywords. More like how a match is found.

1. keywords are case insensitive
2. the name contains any keyword user inputted